### ISSUE TITLE
Do not allow the module to be installed in core versions greater than or equal to 1.23.0

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/entityreference#36]
Now that the module has been merged into Backdrop core (https://github.com/backdrop/backdrop-issues/issues/1301), we should prevent it from being installed in versions of core that include the core variant of the module.

Once this change is merged, we should also create a new release of the contrib module. I believe that this should happen after September 15th, which is when version 1.23.0 of Backdrop core is scheduled to be released.

**Comment by klonos** [comment]
PR provided here: #37

**Comment by herbdool** [comment]
Thank you @klonos
